networking/interconnect/README.md

```diff
@@ -0,0 +1,99 @@
+﻿# Interconnect Attachment and BGP Session
+
+
+This module allows creation of an InterconnectAttachment (VLAN attachment), a Cloud Router and BGP information that must be configured into the routing stack to establish BGP peering.
+
+## Examples
+
+The module allows the creation of one or more  Interconnect Attachments and related BGP Session.
+
+### Interconnect attachments to achieve 99.9% SLA setup
+
+```hcl
```

Comment:
Examples run through automated tests, please make sure you add the relevant comment at the bottom and it gets picked up by the example testing suite. You can run tests using `pytest`.